### PR TITLE
Pin moto < 5

### PR DESCRIPTION
**Pull request by mwittgen**
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mwittgen** [comment]
@conda-forge-admin, please rerender

**Comment by timj** [comment]
I can't see the bit that pins moto.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -146,7 +146,7 @@ outputs:
         - lsstdesc.coord
         - matplotlib-base !=3.6.0
         - mkl  # [linux64]
-        - moto >=3
+        - moto >=3,<5
```
Comment (mwittgen) [inline recipe/meta.yaml:149]: @timj upper pin was added here. Am I missing something here?

**Review by ktlim — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (timj) [reply recipe/meta.yaml:149]: Right. I should have scrolled further down on my phone.

**Pull request merged**